docs/rules/multiline-ternary.md

```diff
@@ -2,16 +2,16 @@
 pageClass: rule-details
 sidebarDepth: 0
 title: vue/multiline-ternary
-description: Enforce newlines between operands of ternary expressions in `<template>`
+description: Enforce newlines between operands of ternary expressions in `<template>` and `<style>`
```

Comment:
Won't this be overridden by `npm run update`?

Reply:
Yep, where do I need to update it?

Reply:
Done, sorry been very busy